city_metrix/layers/smart_cities_lulc.py

```diff
@@ -11,11 +23,194 @@ def __init__(self, land_cover_class=None, **kwargs):
         self.land_cover_class = land_cover_class
 
     def get_data(self, bbox):
-       raise NotImplementedError()
+        crs = get_utm_zone_epsg(bbox)
 
+        # ESA reclass and upsample
+        esa_world_cover = EsaWorldCover().get_data(bbox)
 
+        reclass_map = {
```

Comment:
I think OSM Open Space layer you moved here is also used for a different indicator, so I think you should just use that layer here instead of deleting it

Reply:
Updated OSM layer to have predefined land use types to be reused